ni-libraries/FileLocations.txt

```diff
@@ -0,0 +1,10 @@
+libFRC_NetworkCommunication.so - \usr\local\frc\lib
```

Comment:
This is a list of all ni library files and their location within the image to make finding updates in the future easier